personal/privacyidea/class_mfaAccount.inc

```diff
@@ -935,23 +935,23 @@ class mfaAccount extends plugin
             "plProperties"   => $properties,
             "plRequirements" => array ('onFailureDisablePlugin' => array (get_class())),
             "plProvidedAcls" => array (
-                "allowedTokenTypes"     => _("MFA token types"),
-                "mfaRequiredByRule"     => _("Require MFA due to organizational policy"),
-                "mfaRequiredByUser"     => _("Require MFA due to user preference"),
-                "manageTokens"          => _("Manage tokens"),
-                "addTokenAny"           => _("Add any type of token"),
-                "addTokenPaper"         => _("Add TAN list"),
-                "addTokenRegistration"  => _("Add Recovery key"),
-                "addTokenWebAuthn"      => _("Add Security key"),
-                "addTokenTotp"          => _("Add Authentication app (TOTP)"),
-                "tokenDescription"      => _("Description"),
-                "tokenLastUsed"         => _("Last use"),
-                "tokenFailCount"        => _("Failed login counter"),
-                "tokenStatus"           => _("Status"),
-                "tokenCountAuth"        => _("Authentication count"),
-                "tokenCountAuthSuccess" => _("Successful authentication count"),
-                "tokenRevocation"       => _("Revocation allowed"),
-                "tokenRemoval"          => _("Removal allowed"),
+                "allowedTokenTypes"     => _("MFA token types"),                          #rw
+                "mfaRequiredByRule"     => _("Require MFA due to organizational policy"), #rw
+                "mfaRequiredByUser"     => _("Require MFA due to user preference"),       #rw
+                "manageTokens"          => _("Manage tokens"),                            #rw
+                "addTokenAny"           => _("Add any type of token")         . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenPaper"         => _("Add TAN list")                  . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenRegistration"  => _("Add Recovery key")              . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenWebAuthn"      => _("Add Security key")              . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenTotp"          => _("Add Authentication app (TOTP)") . "&nbsp;(" . _("write only") . ") ", # w
+                "tokenDescription"      => _("Description"),                              #rw
+                "tokenLastUsed"         => _("Last use"),                                 #rw
+                "tokenFailCount"        => _("Failed login counter"),                     #rw
+                "tokenStatus"           => _("Status"),                                   #rw
+                "tokenCountAuth"        => _("Authentication count"),                     #rw
+                "tokenCountAuthSuccess" => _("Successful authentication count"),          #rw
```

Comment:
Please mark (as already discussed in our last video call) the ACLs that are theoretically read-write in GOsa², but read-only by PI API.

Reply:
Please have a look again.